main_helper/cross_server.py

```
@@ -225,6 +225,8 @@ async def maintain_connection(chat_history, lanlan_name):
                                     logger.warning(f"[{lanlan_name}] 发送到analyzer超时")
                                 except Exception as e:
                                     logger.warning(f"[{lanlan_name}] 发送到analyzer失败: {e}")
+                                
+                                # Turn end时不保存聊天记录，只在session end或renew session时保存
```

Ruff (0.14.6)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)